provisioning/roles/publicdb/templates/nginx-default.conf

```diff
@@ -0,0 +1,29 @@
+# ngnix configuration
```

Comment:
Use this generator to get some good settings for configuring SSL for Nginx:
https://mozilla.github.io/server-side-tls/ssl-config-generator/?server=nginx-1.10.3&openssl=1.0.1e&hsts=no&profile=modern

Do not use HSTS (to still allow http), use the Modern profile (do not support older broswers), set the correct nginx/openssl versions for possibly a slightly more accurate config.

Reply:
Nice! Thanks